eng/common/pipelines/templates/steps/set-default-branch.yml

```diff
@@ -4,7 +4,12 @@ parameters:
 steps:
 - pwsh: |
     $setDefaultBranch = (git remote show ${{ parameters.RemoteRepo }} | Out-String) -replace "(?ms).*HEAD branch: (\w+).*", '$1'
+    if ($LASTEXITCODE -ne 0) {
+      Write-Host "Not able to fetch the default branch from git command. Set to master."
+      $setDefaultBranch = 'master'
+    }
     Write-Host "Setting DefaultBranch=$setDefaultBranch"
     echo "##vso[task.setvariable variable=DefaultBranch]$setDefaultBranch"
```

Comment:
nit: echo is just an alias for Write-Host and given you are using Write-Host in other places it would be nice to use it here as well.